extensions/sat/examples/example-sentinel1.json

```diff
@@ -0,0 +1,148 @@
+{
+  "stac_version": "1.0.0-beta.2",
+  "stac_extensions": [
+    "checksum",
+    "sar",
+    "sat"
+  ],
+  "id": "S1A_IW_SLC__1SDV_20150305T051937_20150305T052005_004892_006196_ABBB",
+  "type": "Feature",
+  "bbox": [
+    9.768469,
+    40.479584,
+    13.182633,
+    42.516747
+  ],
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          12.756503,
+          40.479584
+        ],
+        [
+          9.768469,
+          40.877575
+        ],
+        [
+          10.115331,
+          42.516747
+        ],
+        [
+          13.182633,
+          42.119347
+        ],
+        [
+          12.756503,
+          40.479584
+        ]
+      ]
+    ]
+  },
+  "properties": {
+    "title": "Sentinel-1A SLC IW SDV T95 2015-03-05T05:19:37/2015-03-05T05:20:05",
+    "datetime": null,
+    "start_datetime": "2015-03-05T05:19:37.9237880Z",
+    "end_datetime": "2015-03-05T05:20:05.2359650Z",
+    "platform": "sentinel-1a",
+    "constellation": "sentinel-1",
+    "instruments": [
+      "c-sar"
+    ],
+    "sat:platform_serial_identifier": "2014-016A",
+    "sat:orbit_state": "descending",
+    "sat:absolute_orbit": 4892,
+    "sat:relative_orbit": 95,
+    "sat:anx_datetime": "2015-03-05T04:41:46.5788740Z",
+    "sat:across_track_incidence_angle": 33.05, 
+    "sar:instrument_mode": "IW",
+    "sar:polarizations": [
+      "VV",
+      "VH"
+    ],
+    "sar:swaths": [
```

Comment:
That field is not part of the SAR extension and should use a different prefix or be removed, same for all assets.

Reply:
I'm unresolving this as there are still sar:swaths in the file.

Reply:
I don't know how exactly works this review mechanism but there is no more swath in the latest commit

Reply:
Not sure what happened, but now it's gone for me, too.